Timestamp,Operator,URL,Issue
2023-09-01T03:14:26.967042+02:00,KT_SG,https://services.geo.sg.ch/wss/service/SG00002_WMS/guest?REQUEST=GetCapabilities,"HTTPSConnectionPool(host='services.geo.sg.ch', port=443): Max retries exceeded with url: /wss/service/SG00002_WMS/guest?REQUEST=GetCapabilities (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3a8dde6460>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
2023-09-01T03:14:36.984025+02:00,KT_SG,https://services.geo.sg.ch/wss/service/SG00002_WMTS/guest?REQUEST=GetCapabilities,"HTTPSConnectionPool(host='services.geo.sg.ch', port=443): Max retries exceeded with url: /wss/service/SG00002_WMTS/guest?REQUEST=GetCapabilities (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3a8dde6b80>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
2023-09-01T03:14:47.005730+02:00,KT_SG,https://services.geo.sg.ch/wss/service/SG00135_WMS/guest?REQUEST=GetCapabilities,"HTTPSConnectionPool(host='services.geo.sg.ch', port=443): Max retries exceeded with url: /wss/service/SG00135_WMS/guest?REQUEST=GetCapabilities (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3a8dde8430>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
2023-09-01T03:14:57.026356+02:00,KT_SG,https://services.geo.sg.ch/wss/service/SG00135_WMTS/guest?REQUEST=GetCapabilities,"HTTPSConnectionPool(host='services.geo.sg.ch', port=443): Max retries exceeded with url: /wss/service/SG00135_WMTS/guest?REQUEST=GetCapabilities (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3a8dde8d30>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
2023-09-01T03:15:07.124457+02:00,KT_SG,https://services.geo.sg.ch/wss/service/SG00155_WMS/guest?REQUEST=GetCapabilities,"HTTPSConnectionPool(host='services.geo.sg.ch', port=443): Max retries exceeded with url: /wss/service/SG00155_WMS/guest?REQUEST=GetCapabilities (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3a8dde06a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
2023-09-01T03:15:17.145542+02:00,KT_SG,https://services.geo.sg.ch/wss/service/SG00155_WMTS/guest?REQUEST=GetCapabilities,"HTTPSConnectionPool(host='services.geo.sg.ch', port=443): Max retries exceeded with url: /wss/service/SG00155_WMTS/guest?REQUEST=GetCapabilities (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3a8dde85e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
2023-09-01T03:15:27.161515+02:00,KT_SG,https://services.geo.sg.ch/wss/service/SG00046_WMS/guest?REQUEST=GetCapabilities,"HTTPSConnectionPool(host='services.geo.sg.ch', port=443): Max retries exceeded with url: /wss/service/SG00046_WMS/guest?REQUEST=GetCapabilities (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3a8dde6580>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
